src/main/content/_config.yml

```diff
@@ -34,8 +34,7 @@ plugins:
   - jekyll-feed
   - jekyll-asciidoc
   - jekyll-assets
-  - ol-asciidoc
```

Comment:
We cannot remove this custom gem `ol-asciidoc`.  We will need to add it back @ellenwyllie